ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1160,5 +1160,14 @@ literals:
 - return_air_specificenthalpy_sensor
 - compressor_current_sensor
 
+## New Fields For ACUs
+- zone_air_override_temperature_command
```

Comment:
what is this and why is it needed? was not resolved in [issue 502](https://github.com/google/digitalbuildings/issues/502) and i still do not understand how it works, or why it is required. i am also not convinced it is a command.